kornia/feature/dedode/dedode.py

```diff
@@ -174,16 +175,16 @@ def from_pretrained(
     ) -> Module:
         r"""Loads a pretrained model.
 
-        Depth model was trained using depth map supervision and is slightly more precise but biased to detect keypoints
-        only where SfM depth is available. Epipolar model was trained using epipolar geometry supervision and
-        is less precise but detects keypoints everywhere where they are matchable. The difference is especially
-        pronounced on thin structures and on edges of objects.
-
         Args:
-            detector_weights: The weights to load for the detector. One of 'L-upright', 'L-C4', 'L-SO2'.
+            detector_weights: The weights to load for the detector.
+                One of 'L-upright' (original paper, https://arxiv.org/abs/2308.08479),
+                'L-C4', 'L-SO2' (from steerers, better for rotations, https://arxiv.org/abs/2312.02152),
+                'L-C4-v2' (from dedode v2, better at rotations, less clustering, https://arxiv.org/abs/2404.08928)
+                Default is 'L-upright', but perhaps it should be 'L-C4-v2'?
```

Comment:
for me it should be the best/newest one, cc @ducha-aiki

Reply:
I wasnt sure about kornias backward compat, e.g., if someone had v1 before it could be a suprise that suddently its v2 when they update kornia